flows/vision-zero/cris_import/cris_import.py

```diff
@@ -264,7 +266,8 @@ def pgloader_csvs_into_database(directory):
     );
 $$;\n""")
                 cmd = f'pgloader {command_file}'
-                os.system(cmd)
+                if os.system(cmd) != 0:
+                  logger.info('Unable to execute pgloader command')
```

Comment:
I think that is a great idea! I was thinking that we would want subsequent attempts to happen, but you are right - this kind of issue makes them all fail so it should stop when we see the error. I'm going to make this update in the morning. 🙏

Reply:
Mike, I think your idea about giving it multiple tries is totally a good one. You can [decorate the task](https://docs-v1.prefect.io/api/latest/core/task.html#task-2) with a directive on how many retries it's allowed to make, and it will do that, and only after the last attempt fails, will it let the whole flow fail. 

`max_retries=None, retry_delay=None` are the defaults, and maybe a setting of `2` and `60`?